iOS_Concepts.playground/Contents.swift

```diff
@@ -1,3 +1,14 @@
-import UIKit
+import Foundation
 
-var greeting = "Hello, playground"
+
+func compare<T>(a: T, b: T) where T: Equatable & CustomStringConvertible{
```

Comment:
what is `CustomStringConvertible` and what does it do?

Reply:
CustomStringConvertible is a protocol that specifies how to convert a type to a String type. when using it you are telling the compiler that the type can be converted to a String using the description method, the method returns a string that represents a type.